<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBase</class>
 <widget class="QMainWindow" name="DataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>BASE TABLE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>12</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>SORT TABLE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView_2"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Open data base</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_row_box">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Add row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_row_box">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Delete row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reselect_box">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Reselect table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="request">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Make request</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submit_box">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="revert_box">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Revert</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>10</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>CHANGE TABLE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Mathcad UniMath</family>
          <pointsize>10</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>FILTER PATTERN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="dialog_search">
          <property name="font">
           <font>
            <family>Mathcad UniMath</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Boutton_clear">
          <property name="font">
           <font>
            <family>Mathcad UniMath</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
